tests/unit/Transaction.test.ts

```diff
@@ -47,13 +47,7 @@ describe('unit', () => {
           CONTRACT_ADDRESS,
           'CurrencyNetwork',
           'transfer',
-          [
-            COUNTER_PARTY_ADDRESS,
-            '10000',
-            '0',
-            [COUNTER_PARTY_ADDRESS],
-            EXTRA_DATA
-          ]
+          ['10000', '0', [COUNTER_PARTY_ADDRESS], EXTRA_DATA]
```

Comment:
the path should be [USER_ADDRESS, COUNTER_PART_ADDRESS]

Reply:
:heavy_check_mark: